#!/usr/bin/python

#This file just contains our user defined exception for the resourceManager


class ResourceError(StandardError):
  pass

